### PR TITLE
CANS-843 Alert on clicking Use previous ratings

### PR DESCRIPTION
**Pull request by KShresthaCwds**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-843

## Description
Added Alert on clicking Use previous ratings since last CANS assessment

## Tests
- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/AssessmentContainer.js [hunk 1]
```diff
@@ -57,6 +60,8 @@ export default class AssessmentContainer extends Component {
       completeScrollTarget: 0,
       isUnsaved: false,
       isReassessmentModalShown: false,
+      isShowReassessmentAlert: false,
+      isClosed: false,
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainer.js]: This seems unnecessary.

Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentContainer.js]: yeah. just removed it.

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js [hunk 2]
```diff
@@ -1048,4 +1048,20 @@ describe('<AssessmentContainer />', () => {
...
+    describe('render', () => {
+      it('reassessment alert', () => {
+        const alert = shallow(<AssessmentContainer {...defaultProps} />)
+        alert.instance().setState({ isShowReassessmentAlert: true })
+        expect(alert.find('CloseableAlert').exists()).toBe(true)
+        expect(alert.find('CloseableAlert').props().type).toBe('info')
+        expect(alert.find('CloseableAlert').props().isCloseable).toBe(true)
+        expect(alert.find('CloseableAlert').props().message).toBe(
+          'If this child has turned 6 years old since the last assessment, only the common domain items for both age groups will carry over.'
+        )
```
Comment (AK56) [inline app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: I would split the test cases here: one for rendering of `<CloseableAlert>` and a separate one for the message prop

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js [hunk 3]
```diff
@@ -1048,4 +1048,20 @@ describe('<AssessmentContainer />', () => {
...
   })
+
+  describe('<CloseableAlert>', () => {
+    describe('render', () => {
+      it('reassessment alert', () => {
+        const alert = shallow(<AssessmentContainer {...defaultProps} />)
+        alert.instance().setState({ isShowReassessmentAlert: true })
+        expect(alert.find('CloseableAlert').exists()).toBe(true)
+        expect(alert.find('CloseableAlert').props().type).toBe('info')
+        expect(alert.find('CloseableAlert').props().isCloseable).toBe(true)
```
Comment (AK56) [inline app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: nothing seem to drive `isCloseable` prop value and seems to be only set to true, so might be unnecessary here

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js [hunk 4]
```diff
@@ -1048,4 +1048,20 @@ describe('<AssessmentContainer />', () => {
       })
     })
   })
+
+  describe('<CloseableAlert>', () => {
+    describe('render', () => {
+      it('reassessment alert', () => {
+        const alert = shallow(<AssessmentContainer {...defaultProps} />)
```
Comment (AK56) [inline app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: `wrapper` would be a better name for a shallow `<AssessmentContainer>` component

Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: yeah, you are right. will do it.

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js (hunk 2 above)
Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: Done.

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js (hunk 3 above)
Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: Was using isCloseable props so was making sure to just include a test. But I took your suggestion and removed it. Thanks.

File: app/javascript/Application/components/Assessment/AssessmentContainer.test.js (hunk 4 above)
Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentContainer.test.js]: Done

**Comment by KShresthaCwds** [comment]
Included test for isShowReassessmentAlert to be true. Thanks @denys-davydov

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Pull request merged**